ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_snapshot_collection.py

```diff
@@ -177,18 +178,19 @@
 def create_snapcoll(
         client_obj,
         snapcoll_name,
+        volcoll_name,
         **kwargs):
 
     if utils.is_null_or_empty(snapcoll_name):
         return (False, False, "Create snapshot collection failed. snapshot collection name is not present.", {})
     try:
-        snapcoll_resp = client_obj.snapshot_collections.get(id=None, name=snapcoll_name)
+        snapcoll_resp = client_obj.snapshot_collections.get(id=None, name=snapcoll_name, volcoll_name=volcoll_name)
         if utils.is_null_or_empty(snapcoll_resp):
             params = utils.remove_null_args(**kwargs)
             snapcoll_resp = client_obj.snapshot_collections.create(name=snapcoll_name, **params)
-            return (True, True, f"Created snapshot collection '{snapcoll_name}' successfully.", {})
+            return (True, True, f"Created snapshot collection '{snapcoll_name}' inside volume collection '{volcoll_name}' successfully.", {})
```

Comment:
nit: Created snapshot collection '{snapcoll_name}' for volume collection '{volcoll_name}' successfully